### PR TITLE
Added a form extender to also display the checkbox to display to the …

### PR DESCRIPTION
**Pull request by ale-rt**
…session tool

### CODE REVIEW

**Comment by ale-rt** [comment]
This for the moment only logs a line.
Storing the data depends on how https://github.com/euphorie/osha.oira/pull/164 goes.

It is up top you team to decide if you want to merge it right away and an issue in backlog or if you want to hold this one until #164 is merged.

**Review by reinhardt — APPROVED** [review]

**Pull request merged**
